scripts/installers/setup_os2s_decoders.py

```diff
@@ -0,0 +1,118 @@
+"""Script to build and install decoder package."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import glob
```

Comment:
We need to add how and when users need to use these new scripts in the documentations.
Also added more info on the new features it supports like punctuation, aggregated tokenizer, etc.

Reply:
added